cesium/internal/index/domain.go

```diff
@@ -125,10 +124,9 @@ func (i *Domain) Distance(
 				startToFirstEnd.Lower+gap,
 				startToFirstEnd.Upper+gap,
 			)
-			domainBounds.Upper = iter.Position()
+			alignment = telem.NewAlignmentPair(iter.Position(), uint32(iter.Len()/8))
```

Comment:
Why is the sample index here an eight of the domain length? Could you leave a comment here to clarify